cart/domain/cart/services.go

```diff
@@ -0,0 +1,15 @@
+package cart
+
+import (
+	"context"
+
+	"flamingo.me/flamingo-commerce/v3/product/domain"
+)
+
+type (
+	// MaxQuantityRestrictor returns the maximum qty allowed for a given product and cart
+	MaxQuantityRestrictor interface {
+		// Restrict must return the maximum allowed qty or `int(^uint(0) >> 1)` for infinity
```

Comment:
Hm, maybe a

```go
math.MaxInt32
```
would also be a bit more readable on this comment line

Reply:
Yes, your right. Missed that spot when I switched to the math constant. I'll updated the comment accordingly.